adal/build.gradle

```diff
@@ -22,6 +22,7 @@ android {
 
     defaultConfig {
         minSdkVersion 14
+        //noinspection OldTargetApi
         targetSdkVersion 23
```

Comment:
why cannot we put this in the lint suppression? I asked the question in the previous iteration, can you guys help me understand why this cannot be put in the lint suppression rule?